oak_functions_launcher/tests/integration_test.rs

```diff
@@ -29,19 +29,10 @@ lazy_static! {
     };
 }
 
-/// Whether to skip the test. For instance, GitHub Actions does not support KVM, so we cannot run
-/// tests that require nested virtualization.
-fn skip_test() -> bool {
-    std::env::var("OAK_KVM_TESTS")
-        .unwrap_or_default()
-        .to_lowercase()
-        == "skip"
-}
-
 // Allow enough worker threads to collect output from background tasks.
 #[tokio::test(flavor = "multi_thread", worker_threads = 3)]
 async fn test_launcher_virtual() {
-    if skip_test() {
+    if xtask::testing::skip_test() {
```

Comment:
This feels a bit hacky. Skimming the Rust book, we could mark the test as `#[ignored]` and force the ignored tests to be executed with `cargo test -- --ignored`, but I'm not entirely convinced that's much better.

(It'd be nice if we could tag this as `"requires_kvm"` or something -- maybe with features, assuming `cargo test` doesn't enable all features by default?)

Reply:
Yeah i even tried using a custom config to conditionally ignore the test, but it didn't work (I may try that again). OTOH if we decide to move all the testing to kokoro, we may remove this entirely.